README.md

```diff
@@ -612,6 +614,21 @@ SAGEMAKER_TFS_NUM_BATCH_THREADS="16"
 SAGEMAKER_TFS_MAX_ENQUEUED_BATCHES="10000"
 ```
 
+## Other Configurable Environment Variables
+The following environment variables can be set on a SageMaker Model or Transform Job if further configuration is required:
+
+```bash
+# Configures the logging level for GUnicorn.
+# Valid values can be found here: https://docs.gunicorn.org/en/stable/settings.html#loglevel
+# Defaults to "info"
+SAGEMAKER_GUNICORN_LOGLEVEL="debug"
+
+# Configures how long to wait in seconds for GUnicorn
+# to finish starting up before timing out.
+# Defaults to 30.
+SAGEMAKER_GUNICORN_SETUP_TIMEOUT_SECONDS="60"
```

Comment:
Add a link to the actual option of gunicorn. https://docs.gunicorn.org/en/stable/settings.html#timeout
Example: You can mention something like `for setting this parameter, refer: https://docs.gunicorn.org/en/stable/settings.html#timeout`